distributed/deploy/local.py

```diff
@@ -54,6 +54,9 @@ class LocalCluster(Cluster):
         Tornado gen.coroutines.  This should remain False for normal use.
     kwargs: dict
         Extra worker arguments, will be passed to the Worker constructor.
+    blocked_handlers: List[str]
+        A list of strings specifying a blacklist of handlers to disallow on the Scheduler,
+        e.g., ``['feed', 'run_function']``
```

Comment:
Copy-editors sometimes scold me for using `e.g.` rather than `like`, which they say is more easily understood by non-native speakers.

Reply:
updated